patient-service-layer/src/test/java/hu/kuncystem/patient/servicelayer/user/DefaultUserManagerTest.java

```diff
@@ -0,0 +1,58 @@
+package hu.kuncystem.patient.servicelayer.user;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import hu.kuncystem.patient.pojo.user.User;
+import hu.kuncystem.patient.servicelayer.utilities.Hash;
+
+/**
+ * Testing the DefaultUserManager methods
+ *
+ * @author Csaba Kun <kuncy88@gmail.com>
+ * @date 2017. dec. 28.
+ * 
+ * @version 1.0
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "/beans_settings.xml" })
+@ActiveProfiles("live")
+public class DefaultUserManagerTest {
+    @Autowired
+    private UserManager userManager;
+
+    @Test
+    public void start() {

```

Comment:
Instead of having one start you should add several more. For instance:
shouldCreateUserSuccessfullyWhenUserDidNotExist, shouldNotCreateUserWhenUserExist, shouldNotUpdatePasswordIfOldPasswordDoesNotMatch

Something like this. Do not be afraid of long test method names. This helps the reader to understand what's being tested